src/components/Typography/UL.tsx

```diff
@@ -1,10 +1,12 @@
 import styled, { css } from "styled-components";
 
 import { box, BoxProps } from "../Box";
+import { FlexProps } from "../Flex";
 
-type ULProps = BoxProps & {
-  $reset?: boolean;
-};
+type ULProps = BoxProps &
```

Comment:
I'm not sure the UL & LI  components are worthwhile as they so often end up being flexboxes in disguise. A more transparent approach is to use flexes and set the roles to "list" and "listitem" where appropriate.